pkg/sshutil/sshutil.go

```diff
@@ -77,7 +115,17 @@ func SSHArgs(instDir string) ([]string, error) {
 	if len(controlSock) >= osutil.UnixPathMax {
 		return nil, errors.Errorf("socket path %q is too long: >= UNIX_PATH_MAX=%d", controlSock, osutil.UnixPathMax)
 	}
+	configDir, err := store.LimaConfigDir()
+	if err != nil {
+		return nil, err
+	}
+	privateKeyPath := filepath.Join(configDir, filenames.UserPrivateKey)
+	_, err = os.Stat(privateKeyPath)
+	if err != nil {
+		return nil, err
+	}
 	args := []string{
+		"-i", privateKeyPath,
```

Comment:
Question: would it be possible to let ssh read both `~/.lima/_config` keys and `~/.ssh` keys?

Otherwise this will break existing instances.

Reply:
> Question: would it be possible to let ssh read both `~/.lima/_config` keys and `~/.ssh` keys?

Yes, you can specify multiple `-i` options, and the identities will be tried in sequence. I think the server may be limiting the number of attempts to 3, so it could still fail if the user added multiple keys after creating the instance. But that is already a problem with the current implementation, so doesn't change anything. Just another reason to have a lima-specific identity, and always specify that one first.

Reply:
> Question: would it be possible to let ssh read both `~/.lima/_config` keys and `~/.ssh` keys?

I've pushed the change in a separate commit for easier review; lmk if you want me to squash it!